server/build.gradle.kts

```diff
@@ -128,3 +133,31 @@ publishing {
         }
     }
 }
+
+/**
+ * Disables unnecessary publishing tasks:
```

Comment:
```
Configures the publishing, so that:

1. The fat JAR artifact appears only in the remote Maven repository,
which is GitHub Packages in this case.

2. `server` module is only published locally,
since it is needed for the project assembly.

We use `afterEvaluate` because
the publishing tasks to modify are only available at this phase.
```